iFixit/react-commerce
# Add product-list breadcrumbs structured data

connects #104 

This PR adds the breadcrumbs structured data to product-list pages

### QA

1. Visit Vercel preview
2. Verify that product-list are now including a valid breadcrumb ld+json
